.github/workflows/virusScanAllAddons.yml

```diff
@@ -57,35 +56,105 @@ jobs:
             const fs = require('fs')
             const addonsWithoutVT = fs.readFileSync('addonsWithoutVT.txt', 'utf-8').split('\n').filter(Boolean)
             setVirusTotalAnalysisStatus({core}, addonsWithoutVT.slice(0, ${{ env.BATCH_SIZE }}))
-      - name: Create PR for updated VT urls
+      - name: Push updated VT urls
         shell: bash
         run: |
           git add addons
           git config user.name "github-actions"
           git config user.email "github-actions@github.com"
-          git commit -m "Add VirusTotal review URLs"
+          git commit -m "Add VirusTotal results"
+          git pull --rebase
           git push --set-upstream origin ${{ env.BRANCH_NAME }}
-          gh pr create \
-          --title "Add VirusTotal review URLs" \
-          --base ${{ github.ref }} \
-          --head ${{ env.BRANCH_NAME }} \
-          --body "Add VirusTotal review URLs to add-ons"
-          gh pr merge --merge "${{ env.BRANCH_NAME }}"
         env:
           GH_TOKEN: ${{ github.token }}
-      - name: Upload results
-        id: uploadResults
-        if: failure()
-        uses: actions/upload-artifact@v4
+
+  codeQL-analysis-prep:
+    runs-on: windows-latest
+    outputs:
+      addonsWithoutCodeQL: ${{ steps.getAddonsWithoutCodeQL.outputs.addonsWithoutCodeQL }}
+    env:
+      BATCH_SIZE: 10
```

Comment:
temporarily adjusted for testing, we can probably up this to 50-100 once testing is complete